packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -123,6 +135,8 @@ export function DatePicker({
         onCalendarOpen={handleCalendarOpen}
         onCalendarClose={handleCalendarClose}
         dateFormat={["P", "PP", "PPP", "MMM dd yyyy", "MMMM dd yyyy"]}
+        highlightDates={highlightDates}
+        onMonthChange={handleMonthChange}
```

Comment:
can just skip the function and go straight to 
```suggestion
        onMonthChange={onMonthChange}
```

right?

Reply:
Yeah, that should be fine; was just mimicking what was done with `onChange`/`handleChange`. Which...while we're here, should that one get cleaned up too?

Reply:
that would be great! less code to do some run around the code

Reply:
Updated the `onMonthChange` handler. Tried to cleanup `onChange`/`handleChange`, but it turns out the ReactDatePicker gives both date and an event as parameters. Which theoretically should be a non-breaking change, but that breaks tests both here (DatePicker and InputDate) and would probably cause downstream breaks in tests (when others write something like `expect(onChange).toHaveBeenCalledWith(date)` in their component tests).

Reply:
Oh yah, i knew there was a reason why we did it!